RFC-0039-cuda-support.md

```diff
@@ -0,0 +1,197 @@
+
+# [CUDA version support]
+
+**Authors:**
+* @atalman @malfet @tinglvv @nWEIdia @ptrblck
+
+
+## **Motivation**
+
+The proposal is to provide two main benefits
```

Comment:
I am a bit curious why we didn't go for "prodictability" as a motivation here?
Given that cuda versions are released at a relatively stable cadence, why wouldn't we want to align to that (with some delay) to enable partners to predict when these upgrades will happen?

(note that I'm not arguing we should do this, just want to hear the reasoning on your end)

Reply:
Maybe @ptrblck can also chime in for this. I believe for CUDA version updates we would like to first evaluate if update is necessary and there are no major regressions in new version of CUDA, this is why we have a delay and evaluation step. For example we added support for CUDA 12.4, 12.6 and 12.8, skipping 12.5 and 12.7 versions.